src/main/resources/build.template.properties

```diff
@@ -19,17 +19,12 @@ app.region=us-central1
 # e.g. app.version = 8-0-0
 app.version = 8-0-0
 
-# This is the URL for the front-end dev server.
-# This should be used only during development mode where the dev server needs to be whitelisted for CORS.
-app.frontenddev.url=http\://localhost:4200
-
-# This is the port to connect with local Datastore emulator.
-app.localdatastore.port = 8484
-
-# This indicates whether task queues are active (e.g. items added to task queue will be queued for execution).
-# This flag is only used during development mode; in production, task queue will always be active.
-# In addition, during development mode, there is no "queueing", i.e. all tasks will be immediately executed.
-app.taskqueue.active=true
+# This is the URL for the front-end which the user is going to access.
+# This affects all URLs sent in outbound emails.
+# If not explicitly set:
+#   when using GAE, the default domain name is https://{app_id}.appspot.com
+#   when running locally, the default domain name is http://localhost:8080
+app.frontend.url=
```

Comment:
I just remembered, an empty value for a property will not trigger the default value to be used. So for this particular property, the entire thing should be commented out instead:
```suggestion
# app.frontend.url=
```